molecule/provisioner/ansible.py

```diff
@@ -316,9 +317,15 @@ def default_config_options(self):
 
     @property
     def default_options(self):
-        d = {
-            'skip-tags': 'molecule-notest,notest',
-        }
+        if self._config.action == 'idempotence':
+            d = {
```

Comment:
Shall we rename `d` to `options`?

Reply:
I just went with the original. The 'd' var is used in several places in the file.
Therefor i would prefer not to refactor the code in this PR.
Beside that i do not have any preferences here. The code is small enough to not lose context and 'd' is kept local to the functions. Just a temp var which holds a dictionary.

Reply:
:+1: